lib/app.tsx

```diff
@@ -127,12 +127,14 @@ export const App = connect(
     }
 
     componentDidMount() {
-      window.electron.receive('appCommand', this.onAppCommand);
-      window.electron.send(
-        'setAutoHideMenuBar',
-        this.props.settings.autoHideMenuBar
-      );
-      window.electron.send('settingsUpdate', this.props.settings);
+      if (isElectron) {
+        window.electron.receive('appCommand', this.onAppCommand);
+        window.electron.send(
+          'setAutoHideMenuBar',
+          this.props.settings.autoHideMenuBar
+        );
+        window.electron.send('settingsUpdate', this.props.settings);
```

Comment:
also acceptable would be to use optional chaining, which I slightly prefer at the moment due to its brevity… (doesn't require `isElectron`)

```ts
window.electron?.receive('appCommand', this.onAppCommand);
window.electron?.send( 'setAutoHideMenuBar', this.props.settings.autoHideMenuBar );
window.electron?.send('settingsUpdate', this.props.settings);
```

the tradeoff could be argued as losing the context that this is intended to run only in Electron but I feel like that's obvious given the project and also given the naming